main.tf

```
@@ -258,6 +260,7 @@ resource "aws_rds_cluster_instance" "cluster_instance" {
 
   db_subnet_group_name    = "${local.subnet_group}"
   db_parameter_group_name = "${local.parameter_group}"
+  availability_zone       = "${var.instance_availability_zone_list_provided ? element(coalescelist(var.instance_availability_zone_list, list("")), count.index) : ""}"
```

I think we can probably avoid having to have the `instance_availability_zone_list_provided ` variable since we are not going to base any resource counts off the `instance_availability_zone_list` variable.  Instead we can adjust the default for `instance_availability_zone_list` slightly and update this line to 
```suggestion
  availability_zone       = "${element(var.instance_availability_zone_list, count.index)}"
```